src/containers/CustomerServiceCreateForm/CustomerServiceCreateForm.container.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
is this container worth ? actually you only need to put all in the  CreateStaffModal Container and then check which tab is using to call the right API tbh